src/features/DisplayControls/NursingTasks/components/TaskTile.jsx

```diff
@@ -100,7 +102,7 @@ export default function TaskTile(props) {
             <div className="tile-content-subtext">
               <Clock />
               <div>
-                &nbsp;{getTime(administeredTimeInEpochSeconds, startTime)}
+                &nbsp;{enable24Hour? moment(getTime(administeredTimeInEpochSeconds, startTime),"hh:mm").format(timeFormatfor24Hr) : moment(getTime(administeredTimeInEpochSeconds, startTime),"hh:mm").format(timeFormatFor12hr)}
```

Comment:
can we create a utility method in DateTimeUtils like this:
const formatTime(time, inputFormat, outputFormat) => {
    return moment(time, inputFormat).format(outputFormat);
}

As I see, we are using this at multiple places so it's better to create a utility method in DateTimeUtils